### DISCUSSION TITLE
Option to send/ requeue to some other consumer if available on nack() !!

### DISCUSSION THREAD

**Discussion by geminigeek** [cloudamqp/lavinmq#380] [Ideas]
hi,

i am suggesting a feature, this can be a good feature to have , an option on nack() to requeue the msg to some other consumer if available except the one which nack() it , if there is only one consumer do nothing !!

i don't know if its possible already , if yes kindly point me to it 

another way to implement it is with a setting on a consumer to only except a message if the message is redelivering 

thanks for reading

**Comment by baelter** [comment]
Interesting idea.
It's possible today with dead-lettering. But in that case the other consumer have to be on an other queue.

**Comment by carlhoerberg** [comment]
We try to conform to the AMQP 0-9-1 spec and there's no such thing there. Instead as anders suggests, use dead-letter queues. https://lavinmq.com/documentation/dead-letter-exchange
